gfx-memory/fuzz/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "gfx-memory-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
+publish = false
+edition = "2018"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.3"
+arbitrary = { version = "0.4", features = ["derive"] }
+gfx-memory = { path = ".." }
+hal = { package = "gfx-hal", version = "0.5" }
+gfx-backend-empty = { git = "https://github.com/GabrielMajeri/gfx", branch = "hal-0.5" }
+
+[patch.crates-io]
+gfx-hal = { git = "https://github.com/GabrielMajeri/gfx", branch = "hal-0.5" }
+
+# Prevent this from interfering with workspaces
```

Comment:
why shouldn't it be in the project workspace?

Reply:
Seems it can cause issues: https://github.com/rust-fuzz/cargo-fuzz/issues/80